Resources/Контробандист/Прото рольки.yml

```
@@ -0,0 +1,47 @@
+  whitelisted: true
+  startingGear: SmugglerGear
+  alwaysUseSpawner: true
+  hideConsoleVisibility: true
+  canBeAntag: true
+  icon: "JobIconSyndicate"
+  supervisors: job-supervisors-nobody
+  weight: 40
+  displayWeight: 40
+  setPreference: true
+  access:
+  - Syndicate #?
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)